pkg/plan/stream.go

```diff
@@ -86,22 +86,22 @@ func StreamJSON(channel <-chan TrackResponse, device io.Writer, pretty bool) err
 // this function will block execution forever. If the plan failed, it returns
 // the error that made the plan fail.
 func StreamFunc(channel <-chan TrackResponse, function func(TrackResponse)) error {
-	var err = new(multierror.Error)
+	var merr = multierror.NewPrefixed("found")
```

Comment:
I'm not sure I understand why were are using `"found"` ? 🤔 would `"track response"` or `"deployment plan"` not make more sense for the end user?

Reply:
I've corrected the error: